geopyspark/avroregistry.py

```diff
@@ -106,19 +100,15 @@ def tile_encoder(obj):
         return datum
 
     @classmethod
-    def multiband_encoder(cls, obj):
+    def tile_encoder(cls, obj):
+        band_count = obj['data'].shape[0]
 
         def create_dict(index):
```

Comment:
Ok, we should log this as an issue if we're not going to address this right now.